posthog/async_migrations/migrations/0002_events_sample_by.py

```diff
@@ -68,7 +68,7 @@ class Migration(AsyncMigrationDefinition):
     posthog_max_version = "1.33.9"
 
     service_version_requirements = [
-        ServiceVersionRequirement(service="clickhouse", supported_version=">=21.6.0,<21.7.0"),
```

Comment:
cc @yakkomajuri I don't remember why this was added in the first place but looks safe to be removed.

Reply:
I don't think this is still relevant, though I might be missing something due to tests.

This will make it to the next release and be required to be run before then anyways :)